bare_metal_billing/main.py

```diff
@@ -23,14 +23,34 @@ def parse_time_argument(arg):
     return arg
 
 
+def parse_time_range(arg: str):
+    start_str, end_str = arg.split(",")
+    start_time, end_time = [parse_time_from_string(i) for i in (start_str, end_str)]
+    if start_time >= end_time:
+        raise ValueError(f"Start time {start_time} must be before end time {end_time}.")
```

Comment:
you should raise an argparse.ArgumentTypeError here instead of ValueError. Otherwise, it actually won't print the message from ValueError.